pom.xml

```diff
@@ -67,6 +67,8 @@
         <version.plugin.release>3.0.0-M7</version.plugin.release>
         <version.plugin.gpg>3.0.1</version.plugin.gpg>
         <version.plugin.staging>1.6.13</version.plugin.staging>
+        <!--SBOM generation -->
+        <version.plugin.cyclonedx>2.7.9</version.plugin.cyclonedx>
```

Comment:
Is 2.7.9 the right version? There are newer versions available (2.7.11 is the last 2.7.x release, 2.8.0 is also available which updates to CDX 1.5)

Reply:
I see 2.7.9 is what we have on the main branch currently.

Reply:
I will merge this one for now and update the plugin version when the time is closer to the release date so that we can get the latest plugin version. I will check other plugin version then.